### ISSUE TITLE
[Question]: API

### ISSUE THREAD

**Issue by peizimo** [infiniflow/ragflow#3574]
### Describe your problem

![image](https://github.com/user-attachments/assets/a74067a4-a197-466a-a616-3ec7a1f4cc58)

**Comment by KevinHuSh** [comment]
Is that the same issue #3570  ?

**Comment by peizimo** [comment]
YES,thanks,Should we fix it, or should ragflow technicians fix the API
